### DISCUSSION TITLE
Project 1 Feedback

### DISCUSSION THREAD

**Discussion by vikrantdeshpande09876** [airavata-courses/neo#73] [General]
### Discussed in https://github.com/airavata-courses/neo/discussions/48

<div type='discussions-op-text'>

<sup>Originally posted by **vikrantdeshpande09876** February  9, 2022</sup>
Tested out the application on my local machine. Runs fine! 💯 
Here's a few issues I observed:
1. The "edit" feature doesn't seem to work. It's essentially just doing the same thing as the trashcan icon (clearing the workspace).
2. All microservices are allocated the maximum memory, but it might be prudent to set CPU resources LIMITs based on actual functionality of each.
3. The login feature pops up a Google login each time, but shouldn't this be required only while registering? Signing in could be made available based on a static username and password too (if someone doesn't want to link their Google account to this system).

Best,
Team Scapsulators.</div>

**Comment by niravraje** [comment]
Thanks Team Scapsulators!
That's a good point about allocating resources based on microservice functionality. We'll look into our runtime logs to identify our CPU-intensive services. Once identified, we can update our docker-compose YAML file.
We are also aware of these minor UI issues and have identified how to fix them - should be a quick update. Thanks for reviewing!
